stbt/__init__.py

```diff
@@ -501,7 +501,9 @@ def is_screen_black(frame=None, mask=None, threshold=None):
       binary threshold is applied to the frame. The ``threshold`` value is in
       the range 0 (black) to 255 (white). The global default can be changed by
       setting ``threshold`` in the ``[is_screen_black]`` section of
-      :ref:`.stbt.conf`.
+      :ref:`.stbt.conf`. You may need to install the tesseract language pack;
+      see installation instructions `here
+      <https://stb-tester.com/manual/troubleshooting#install-ocr-language-pack>`_.
```

Comment:
This is useful documentation.  We could even have a lint for it, but it probably wouldn't be worth the effort.

Reply:
This is in the wrong docstring.

Reply:
Oops moved that to `stbt.ocr` docstring.